inductiva/molecules/protein_solvation/README.md

```diff
@@ -38,17 +38,19 @@ To test this scenario we have available one example of a protein PDB file - [dow
 
 inductiva.api_key = "YOUR_API_KEY"
 
- # Initialize the scenario
- scenario = inductiva.molecules.ProteinSolvation(
-     protein_pdb = "alanine.pdb",
+# Initialize the scenario
+lysozyme = inductiva.molecules.examples.load_lysozyme()
```

Comment:
How long does it take this simulation in the default machines?

Reply:
It took about 5 minutes.

Reply:
I can make it 0.01 and then it would take 2m, but then the output will have only 10 frames. But I think that for an example its okay.

Reply:
I change the example to be insulin, since it is smaller and now takes 1 minute.